group_vars/rhcs.yml.sample

```diff
@@ -114,6 +114,10 @@ fetch_directory: ~/ceph-ansible-keys
 #ceph_stable_release: dummy
 #ceph_stable_repo: "{{ ceph_mirror }}/debian-{{ ceph_stable_release }}"
 
+#nfs_ganesha_stable: true # use stable repos for nfs-ganesha
```

Comment:
if you add a var in all.yml.sample and then call `./generate_group_vars_sample.sh` the rhcs.yml.sample will automatically be updated.